recipes/gromacs/4.6.5/meta.yaml

```diff
@@ -0,0 +1,27 @@
+about:
+  home: http://www.gromacs.org/
+  license: GNU Lesser General Public License (LGPL)
+  summary: GROMACS is a versatile package to perform molecular dynamics.
+
+package:
+  name: gromacs
+  version: 4.6.5
+
+build:
+  number: 0
+  skip: True #  [osx]
+
+source:
+  fn: gromacs-4.6.5.tar.gz
+  url: ftp://ftp.gromacs.org/pub/gromacs/gromacs-4.6.5.tar.gz
+  md5: 184a537ea52429dbc2a681f98b46ed84
+
+requirements:
+  build:
+    - gcc
+    - cmake
+  run:
```

Comment:
@leobiscassi it's not everytime that you need libgcc, but it's hard to know this before. We will hopefully have better recommendations and better testing soon.
@johanneskoester for example mulled is pretty good in testing these things. But this means we need to put mulled build into travis from this repository.